### PR TITLE
Add information about the identifiers used by Mondo to the README

### PR DESCRIPTION
**Pull request by gaurav**
I noticed that the information on constructing a Mondo identifier is currently located in two places:
1. http://obofoundry.org/id-policy says that the URI `"http://purl.obolibrary.org/obo/" IDSPACE "_" LOCALID` should be used.
2. https://mondo.readthedocs.io/en/latest/editors-guide/f-entities/ says that "All classes are in MONDO ID space"

I really like how the [LOINC User Guide (in section 12.7.2)](https://loinc.org/kb/users-guide/recommendations-for-best-practices-in-using-and-mapping-to-loinc/#using-a-uri-to-identify-loinc-concepts-as-resources-in-rdf) clearly describes how to construct an RDF URI for a LOINC concept, and I think it would be great to include this information in Mondo's documentation somewhere.

In this PR I propose a short paragraph on the main README page in a new section entitled "Identifiers" that would contain this information. If there is a better place to place this document (e.g. the editor's guide linked to above), let me know and I'll make a PR to target that file itself. Feel free to change my proposed wording or move it another file if that would be better.

### CODE REVIEW

**Review by matentzn — CHANGES_REQUESTED** [review]
This content is correct and very much appreciated, but it needs to go into the "Mondo user guide". I can only find an editors guide and a developer guide.

@nicolevasilevsky What would be the best place to put such content? Should we start building a "user guide"?

**Comment by nicolevasilevsky** [comment]
on our readthedocs documentation, we actually have a "user's corner" on the left hand nav, but I have my doubts if many users know that is there.

image

I've always thought the [website](https://mondo.monarchinitiative.org/) was meant to be more of the user documentation. But then there was talk at some point about merging the website and the readthedocs site, which I am in favor of.

I think a more prominent users guide is a great idea. 

At C-Path, we are going to have a week-long hackathon in a few weeks where we are going to focus on documentation. Maybe Mondo should consider a 1 or 2 day hackathon to focus on revising + updating documentation and aggregating and reorganizing all of the docs?

**Review by twhetzel — CHANGES_REQUESTED** [review]
small change requested

**Comment by gaurav** [comment]
My FHIR RDF colleagues pointed out one other thing I missed in addition to Trish's feedback:
- [ ] Add a note either here or on the [MONDO codesystem page on terminology.hl7.org](https://terminology.hl7.org/CodeSystem-MONDO.html) asking users to use MONDO identifiers instead of CURIEs, i.e. when representing MONDO:0005015 "diabetes mellitus" in FHIR, they should use `system=http://purl.obolibrary.org/obo/mondo.owl` and `code=0005015` rather than `code=MONDO:0005015` (see discussion at https://github.com/w3c/hcls-fhir-rdf/issues/127).

File: README.md [hunk 1]
```diff
@@ -9,6 +9,12 @@ This is the repository for managing the Mondo Disease Ontology (Mondo). Mondo ai
  * **Mondo website**: https://monarch-initiative.github.io/mondo
  * **OBO Foundry Mondo page**: http://obofoundry.org/ontology/mondo.html
 
+## Identifiers
+
+Concepts in Mondo are represented by URIs like http://purl.obolibrary.org/obo/MONDO_0005015 or by
+compact URIs with the prefix `MONDO:`, such as `MONDO:0005015`, as required by the
```
[the remark this answers is not in the document]
Comment (gaurav) [reply README.md:15]: How about `... compact URIs ([CURIEs](https://en.wikipedia.org/wiki/CURIE)) with ...`? I like the idea of readers being able to learn more about CURIEs if they're not familiar with the term.

**Comment by gaurav** [comment]
@nicolevasilevsky:
> on our readthedocs documentation, we actually have a "user's corner" on the left hand nav, but I have my doubts if many users know that is there. Screenshot pasted below

I've updated this PR [add an 'Identifiers' page](https://github.com/monarch-initiative/mondo/blob/3a05a22c2daca4f43372030089178a4719f2de6c/docs/editors-guide/identifiers.md) to the Mondo User Guide -- I'm not sure what other content would make sense on that page, but if this approach makes more sense than adding this paragraph to the README, then that would be fine by me too!

**Review by matentzn — APPROVED** [review]
Lets merge this and reconsider it during our Mondo documentation hackathon in June!

**Comment by nicolevasilevsky** [comment]
> Lets merge this and reconsider it during our Mondo documentation hackathon in June!

when is the hackathon in June?

**Comment by matentzn** [comment]
> when is the hackathon in June?

No date yet.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (gaurav) [reply README.md]: @twhetzel Just wanted to check if this change (from 407924f) addressed your concern here, Trish! If not, I'm happy to go with your formulation.

**Comment by gaurav** [comment]
> Lets merge this and reconsider it during our Mondo documentation hackathon in June!

That sounds good to me, but of course I don't get a vote :)

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (twhetzel) [reply README.md]: yes, that looks great!

**Comment by matentzn** [comment]
@gaurav thank you for your patience and contribution here! Really sorry it took so long! We highly appreciate any further suggestions as well :)

**Pull request merged**